style.css

```
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-style: italic;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
```

Vertical and horizontal Scrollbar is visible on your website, that should'nt be the case. U can hide it by using the following code below
```suggestion
  overflow: hidden;
```

---

amazing,
looks great now